sales/customer/openapi.yaml

```diff
@@ -10,12 +10,20 @@ info:
   title: Customer Information Service
 
 servers:
-- url: https://api.kong-air.com
+- url: http://localhost:8083
```

Comment:
@battlebyte are you re-patching these url changes somewhere?  If we change this, then the URLs in the developer portal are not helpful for consumers of the API SDK pages.  For example: https://portal.kong-air.com/spec/451019f3-bd60-4938-a0b5-84e52b4eebbb/2f1404fa-73bf-4a0d-9795-8bf13bc95acf

Reply:
Good point. I changed those so that contract testing / load testing / security testing use the server url. I will patch them before the tests during pipeline execution only.